multithreaded/examm_mt.cxx

```diff
@@ -102,7 +102,11 @@ int main(int argc, char** argv) {
     WeightRules* weight_rules = new WeightRules();
     weight_rules->initialize_from_args(arguments);
 
-    RNN_Genome* seed_genome = get_seed_genome(arguments, time_series_sets, weight_rules);
+    string seed_genome_type;
```

Comment:
Does this get used? Might want to remove it otherwise.

Reply:
snap, I thought I deleted that good catch